### ISSUE TITLE
How to ignore completed tasks that due in the past?

### ISSUE THREAD

**Issue by Eleanor962** [Leonezz/obsidian-tasks-calendar-wrapper#42]
I noticed that there's is a "focus on today" feature. However, I'd like to keep full track to "today" and future tasks, but find history are too distractive.
Just wondering if there's any solution for ignoring completed tasks that due in the past.

**Comment by Leonezz** [comment]
Yes there is an option for ignoring any status of tasks called Hide Tasks of Specific Status. Just enter the corresponding markers and they will be ignored, such as the following setup ignores all tasks with status "cancelled" and "completed":

![image](https://github.com/Leonezz/obsidian-tasks-calendar-wrapper/assets/33564074/f4250f2b-a50d-42a0-ba5a-1b45823d8e3f)

Hope it answers your question!

**Comment by Eleanor962** [comment]
Thanks for the timely reply!
